bazel/linux/providers.bzl

```diff
@@ -77,3 +77,12 @@ RuntimeBundleInfo = provider(
         "check": "list of RuntimeInfo, executable (and its runfiles) to run OUTSIDE the VM AFTER the RUN to check if the run was successful",
     },
 )
+
+BootRamImageInfo = provider(
+    doc = """Maintains the information necessary to represent a BootRam image, needed for aarch64 qemu acf-machine boot.""",
+    fields = {
+        "name": "Name of the rule that defined this bootram image.",
+        "arch": "Architecture this image was built for.",
```

Comment:
looking at the other patch -- I don't see this `arch` field being used either.  I see a check for `if ki.arch == 'aarch64'` and then the `.image` is just used.   I suppose you could leave this `.arch` field here and in the other place verify that `ki.arch == BootRamImageInfo.arch` as a safety check.

Reply:
Added the check.